### ISSUE TITLE
Not work in Xcode 8 beta 5 and 6

### ISSUE THREAD

**Issue by owenzhao** [johnno1962/Refactorator#6]
Will you update or you will wait until Xcode 8 released?

**Comment by johnno1962** [comment]
It’s already updated for Xcode 8. Is there a problem? You’ll need to “unsign” your Xcode to have it load.

**Comment by owenzhao** [comment]
I cloned the project.  Build it in Xcode. Quit Xcode then reopen it. Select load plugin bundle. But I didn't see the refactor menu in right click as I saw in Xcode 7

**Comment by johnno1962** [comment]
Please see this note: https://github.com/alcatraz/Alcatraz/issues/475 & https://github.com/steakknife/unsign

**Comment by owenzhao** [comment]
Thank you. Just another question. If I unsign Xcode, I can't use that Xcode to release apps to App Store. So I should prepare another Xcode? Or the unsign version is still OK to release apps to App Store?

**Comment by johnno1962** [comment]
I don’t know but it wouldn’t suprise me if Xcode needed to be signed for a release to the App store. You can switch unsigning off and on again by exchanging the binaries as needed.

**Comment by golfguy0082** [comment]
Recognizing that I'm completely ignorant of the complexity, is there any possibility of converting this to an editor extension, or is that out of the quesiton?

**Comment by owenzhao** [comment]
No, it is not possible. For current Apple's Xcode extension, you just get a line reader for the current source file. For example, if your project has "foo.swift", "bar.swift", you can only get "foo.swift" unless the user switch to "bar.swift", which means there is no way you could get all the pictures of the whole project automatically, which is fatal for something like `Refactorator`.
